numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -262,7 +262,7 @@ def runF2C(fortran_filename, output_dir):
             ["f2c"] + F2C_ARGS + ['-d', output_dir, fortran_filename]
         )
     except subprocess.CalledProcessError:
-        raise F2CError
+        raise F2CError from None
```

Comment:
This one would probably be better as `from e`

Reply:
Okk I'll do it.

Reply:
What happened here? Was this removed on purpose?

Reply:
Yes it was removed on purpose, I will correct it in another PR.